### ISSUE TITLE
More columns to add to main view

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#92]
- Number of top conference papers
- Sector
- The two workforce numbers we have
- Citations per AI paper
- Number of AI papers in last year of complete data
- 5-year growth in papers
- 5-year growth in patents

**Comment by za158** [comment]
Columns displayed by default should be: company, country, AI pubs, AI patents, workforce tt1

**Comment by brianlove** [comment]
> * Sector

This depends on #120

----

> * Citations per AI paper

My impression has been that `data.articles.citation_counts` has been _all_ citations over all paper types, not just AI papers.  Is `citation_counts` actually AI-specific, or does this column depend on an AI-specific field being added?

----

> * 5-year growth in papers

I can calculate the growth rate, but then we wouldn't have rankings.  If we want rankings in this column (and the similar patents column) then this stat should be pre-calculated and included in the data

----

> * 5-year growth in patents

This depends on #125 adding data for all patents.  There's also the same question as 5-year paper growth regarding rankings

**Comment by brianlove** [comment]
@jmelot The range of years for articles given in `overall.json` is currently `startArticleYear=2019`, `endArticleYear=2022`, which covers 4 years (inclusively).  Should "_5-year growth in papers_" be changed to "_**4-year** growth in papers_", will these ranges be updated in the future, or will these growth columns use a different range than those used in the detail page?

**Comment by za158** [comment]
I don't feel strongly about 4 vs 5 other than that it would be nice to be consistent with Map of Science, which I believe is 5 years. I do not mean to imply that we should be using two different growth ranges within the same tool. @jmelot Do you have thoughts/insight here

**Comment by jmelot** [comment]
Whoops I think I just made a mistake here, I will update in a separate PR

**Comment by brianlove** [comment]
Closed by #154
